src/util/get-all-keys.js

```diff
@@ -0,0 +1,7 @@
+
+export default function keys(obj = {}, { enumerable = false } = {}) {
+  const listOfKeys = Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj);
```

Comment:
True, it's confusing because the first one returns non-enumerable and enumerable. And the second one only enumerable. Will change that for consistency.